airbyte-integrations/connectors/source-gas-prices/source_gas_prices/state_code_list.py

```diff
@@ -0,0 +1,53 @@
+STATE_CODE_LIST = [
```

Comment:
this is used only in one place, maybe keep in the same file

Reply:
I think this might be cleaner to keep in an external file to keep the source.py file clean